terraform/projects/infra-ask-export/main.tf

```
@@ -27,6 +27,12 @@ variable "export_s3_bucket_name" {
   description = "Bucket name to allow write permissions"
 }
 
+variable "create_bucket" {
+  type        = "boolean"
+  description = "Whether to create the bucket, in production we're expecting to use a bucket in a different AWS account"
+  default     = false
```

Since this is ye olde crufty Terraform 0.11, the type system is quite tricksy. There's no boolean type for `variable`; it'll actually use string. The docs also [recommend using string literals "true" and "false" in this context](https://www.terraform.io/docs/configuration-0-11/variables.html#booleans), because TF 0.11 will coerce the bool literal to a string.
```suggestion
  description = "Whether to create the bucket, in production we're expecting to use a bucket in a different AWS account"
  default     = "false"
```

---

Great thanks. Would you mind giving this a quick review: https://github.com/alphagov/govuk-aws-data/pull/942 I merged the govuk-aws-data PR before checking this one 🤦‍♂️